terraform/variables.tf

```diff
@@ -279,6 +279,7 @@ locals {
     "run.googleapis.com/vpc-access-egress" : "private-ranges-only"
   }
   default_service_annotations = {
+    "run.googleapis.com/binary-authorization" : "default"
```

Comment:
I wonder if using this requires "run.googleapis.com/launch-stage" : "ALPHA", let me ask for confirmation

Reply:
All good, I got confirmation that Beta will work.

Reply:
Yea, it applied cleanly on two envs